ebsco_adapter/ebsco_adapter_iceberg/tests/test_mocks.py

```diff
@@ -0,0 +1,392 @@
+import datetime
```

Comment:
These are copied from catalogue-graph and will be consolidated when we merge the projects.